api-samples/identity/identity.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+
+var googleProfileUserLoader = (function() {
```

Comment:
I'd like to make a few style updates to this file:

- Use `let` / `const` instead of `var`
- Just have a named function that is called on `window.addEventListener("load")` instead of assigning a wrapped function like this.
- Remove the `@corecode` comments

I'm happy to do this if you would prefer :)